src/org/opendatakit/briefcase/ui/export/components/ExportFormsTableViewModel.java

```diff
@@ -115,14 +104,15 @@ private void removeConfiguration(FormStatus form) {
     triggerChange();
   }
 
-  private void updateDetailButton(FormStatus form, JButton button) {
-    button.setForeground(form.getStatusHistory().isEmpty() ? LIGHT_GRAY : DARK_GRAY);
+  private void updateDetailButton(FormStatus form, DetailsStatusButton button) {
+    button.setStatus(!(form.getStatusHistory().isEmpty()));
   }
 
-  private void updateConfButton(FormStatus form, JButton button) {
-    button.setForeground(forms.hasConfiguration(form) ? NO_CONF_OVERRIDE_COLOR : DARK_GRAY);
+  private void updateConfButton(FormStatus form, ExportConfigurationButton button) {
+    button.setStatus(forms.hasConfiguration(form));
```

Comment:
I think `button.setStatus()` is kinda unexpected because the meaning of "status" in a conf button is different than the one in a details button. I believe you had to have the same name for both classes because they extend it from `ButtonProcessing`, but this is not really a requirement, since you're using the subclasses (`DetailStatusButton` and `ExportConfigurationButton`) instead of the interface, thus not doing Polymorphism (which is totally fine here, by the way).

This is telling us that the `setStatus` method is not part of the `ButtonProcessing` interface. I'd suggest you remove it from the interface and take the chance to give it a more specific name in `ExportConfigurationButton` e.g. `setConfigured`.

Reply:
Sure.